draft-ietf-webtrans-http3.md

```diff
@@ -76,7 +76,62 @@ server.  An HTTP/3 server is the server that terminates HTTP/3 connections; a
 WebTransport server is an application that accepts WebTransport sessions, which
 can be accessed via an HTTP/3 server.
 
-# Protocol Overview
+# Overview
+
+## QUIC, WebTransport, and HTTP/3
+
+QUIC version 1 {{!RFC9000}} is a secure transport protocol with flow control and
+congestion control. QUIC supports application data exchange via streams;
+reliable and ordered byte streams that can be multiplexed. Stream independence
+can mitigate head-of-line blocking. While QUIC provides streams as a transport
+service, it is unopinionated about their usageage. Applicability of streams is
+described by section 4 of {{?RFC9308}}.
+
+HTTP is an application-layer protocol, defined by "HTTP Semantics" {{!RFC9110}}.
+HTTP/3 is the application mapping for QUIC, defined in {{!RFC9113}}. It
+describes how streams are used to carry control data or HTTP request and
+response message sequences in the form of frames, as well as describing details
+of stream and connection lifecycle management.
+
+WebTransport over HTTP/3 makes it possible for an application to directly access
+QUIC transport via an HTTP/3 connection. This relies on the WebTransport
+handshake (extended CONNECT method) to provide some prior checks. This is
+important or the Web security model where same-origin and cross-origin resource
+access is very important. Post-handshake, QUIC streams use header bytes for
+accounting purposes, but after that an application can use QUIC streams however
```

Comment:
I'm not sure I totally follow this sentence. What is it trying to say?
Is this trying to say that WebTransport uses HTTP/3 to convey header bytes via a QUIC stream, but then eventually just uses the QUIC stream directly?

Reply:
Maybe it's trying to say that a WebTransport application has access to QUIC streams, other than the header bytes?